File: LeetCode/May challenge/Day 4/Day_4.cpp
class Solution {
public:
    
    int findComplement(int num) {
        int res=0,p=0;
        while(num)
        {
            res+= !(num%2)*pow(2,p);
            num/=2;
            p++;
        }
        return res;
    }
};

File: LeetCode/May challenge/Day 4/Day_4.py
class Solution:
    def findComplement(self, num: int) -> int:
        b = bin(num).replace("0b", "")
        new_b = ""
        for i in b:
            if i == "1":
                new_b += "0"
            else:
                new_b += "1"
        return int(new_b, 2)
